pkg/testutil/require.go

```diff
@@ -261,7 +265,11 @@ func requireDownGradeOptionSet(t *testing.T, sb testenv.DiffingSandbox, b testut
 	}
 
 }
-func requireDecommissionNode(t *testing.T, sb testenv.DiffingSandbox, b testutil.ClusterBuilder, numNodes int32) {
+
+// TODO I do not think this is correct.  Keith mentioned we need to check something else.
```

Comment:
we need to check `cockroach node list` to see if the decommissioned node is listed as "decommissioned" and not as something else (like down or failed)

Reply:
Sure, but not in this PR.  Please file an issue

Reply:
https://github.com/cockroachdb/cockroach-operator/issues/535